### ISSUE TITLE
Ara bölümü için.

### ISSUE THREAD

**Issue by Bora185** [tingirifistik/Enough#21]
"Ara" özelliğinin çalışmadığını fark ettim. Nedeni ise, örnek olarak "call.py" dosyasında porsche sitesi için yazılan bölümdeki linke gidildiğinde o linkin aynı bir form gibi çalıştığı anlaşılıyor. Yani anladığım kadarıyla çalışma mantığı şu şekilde: biz numarayı o forma giriyoruz ve site yetkilileri formu gördüklerinde girilen numarayı arıyor. Ondan dolayı anında arama olmuyor. Bunun çözümü ise bazı sitelerde "SMS veya arama ile doğrulama (girilen numaranın doğruluğunu tespit etmek için numarayı çaldırıyor)" özelliği oluyor. O sitelerden arayarak doğrulama API'larını kullanırsak sorun çözülür gibi. Ben bu şekilde sorunu çözmeyi deneyeceğim, çözebilirsem pull request atarım.

**Comment by tingirifistik** [comment]
Evet, özellikle dandik sitelerde keyifleri isterse ararlar gibi bir şey oluyor. 
İlk başta ara özelliği için sadece anında arayan siteleri kullanacaktım ama çok fazla site bulamadım. Eğer bulabilirsen muhteşem olur.

**Comment by 4rcturus** [comment]
bu ozellik icin takipteyim.

**Comment by lotryn** [comment]
> Evet, özellikle dandik sitelerde keyifleri isterse ararlar gibi bir şey oluyor. İlk başta ara özelliği için sadece anında arayan siteleri kullanacaktım ama çok fazla site bulamadım. Eğer bulabilirsen muhteşem olur.

discordunuzu paylaşırmısınız

**Comment by tingirifistik** [comment]
Bio kısmında var.

**Comment by dermany0** [comment]
YOK DİSCORD YAZARMISIN KNKM BEYTAAA BEYAAAA

**Comment by tingirifistik** [comment]
https://www.dunyagoz.com/goz-testleri/

**Comment by dermany0** [comment]
Ahahaha yaptım test kanka ama harbi göremedim la versene bi lütfen ortak çalışalım

**Comment by tingirifistik** [comment]
:) 

tingirifistik#8393

**Comment by dermany0** [comment]
ADAMSIN EKİCEMCKANK EKİCEMCKANKAM EK
EkicEKLC EKLİCEM KANKAM COD X İSMİYLE COD X BEN :);)))
